remill/Arch/Arch.cpp

```diff
@@ -232,27 +214,54 @@ const Register *Arch::RegisterByName(const std::string &name) const {
   }
 }
 
-const Arch *Arch::GetMips(llvm::LLVMContext &, OSName, ArchName) {
+auto Arch::GetMips(llvm::LLVMContext *, OSName, ArchName) -> ArchPtr {
```

Comment:
Lets remove `GetMips` and all MIPS related things while we're at it. We don't support it.